interpreter/syntax/types.ml

```diff
@@ -81,7 +81,7 @@ let match_global_type gt1 gt2 =
 let match_extern_type et1 et2 =
   match et1, et2 with
   | ExternFuncType ft1, ExternFuncType ft2
-  | ExternEventType ft1, ExternEventType ft2 -> match_func_type ft1 ft2
+  | ExternTagType ft1, ExternTagType ft2 -> match_func_type ft1 ft2
```

Comment:
I kept this one as-is for now because the `ExternFuncType` and `ExternTagType` cases here share a right-hand side, so the variable can't be renamed. I could duplicate the RHS code to do this renaming though.

Reply:
Oh I see. Then that's fine, nevermind.